README.md

```diff
@@ -259,10 +259,22 @@ It's often useful to plug some custom steps into the testing environment. Custom
 tests short and self-explanatory and at the same time help teams to add generic assertions that are meaningful in their 
 environment.
 
-To add custom steps in YAKS, you can fork + clone the [yaks extension](https://github.com/nicolaferraro/yaks-extension) repository, that provides
-an example of how to do that.
+To add custom steps in YAKS, you can look at the example provided in the [examples/extension](/examples/extension) directory.
+The example consists of a feature file ([examples/extension/extension.feature](/examples/extension/extension.feature)) using a custom step from a local project 
+([examples/extension/steps](/examples/extension/steps)).
 
-You can add your own steps to that project and follow the instructions in order to install them in the YAKS environment.
+To run the example:
+
+```
+yaks test extension.feature -u steps/
+```
+
+The `-u` flag stands for "upload". The steps project is built before running the test and the artifacts are uploaded to a
```

Comment:
I can not see that the steps project is being built before running the test. Can you please point me to it?

From what I see in Go code the `-u` flag requires the path to the already built `artifact.jar`. Maybe I am missing something

Reply:
The example is provided in the examples/extension dir, with a feature file and a maven project